common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -1476,6 +1476,9 @@ def reset_problem(self, _data):
         # Pull in the new problem seed
         self.set_state_from_lcp()
 
+        # Grade may have changed, so publish new value
+        self.publish_grade()
```

Comment:
So this was a preexisting bug, right? As in, it was broken without robust grading and just would have remained broken?